index.rst

```diff
@@ -4,202 +4,201 @@ Current status of APDB and PPDB implementation
+Data written by the AP pipeline is stored into the regular tables and may optionally be written to this replica chunks tables.
+If replication tables are enabled, the APDB has an additional option to skip writing the data for a ``DiaObject`` not used by the AP pipeline, which saves storage space.
+In this scenario, the table containing the latest version of ``DiaObjects`` would be used by the AP pipeline.
 
 
 Replication process
 -------------------
 
-A separate replication service is responsible for moving recently added APDB data to PPDB.
-APDB writes into its replication tables which are partitioned by the chunk window, which is typically 10 minutes.
-When a chunk is complete it can be transferred to PPDB.
+A dedicated replication service will be responsible for moving recently added APDB data to PPDB.
+The APDB writes into its replication tables, which are typically partitioned by a 10-minute chunk window.
+When a chunk is complete, it can be transferred to the PPDB.
 
-The replication process watches APDB for its list of existing chunks and decides when a new chunk can be transferred to PPDB.
-It also keeps a list of already transferred chunks in a separate table in PPDB.
+The replication process watches the list of existing chunks in the APDB and decides when a new one can be transferred to the PPDB.
+The identifiers of the chunks which have been successfully transferred are cataloged in a separate table within the PPDB.
 
-Replica chunks that have been transferred to PPDB can be deleted from APDB.
-The same replication process is responsible for this cleanup, which happens with some delay (usually a few days).
+Replica chunks that have been transferred to the PPDB can subsequently be deleted from the APDB.
+The same replication process is responsible for this cleanup, which typically happens within a few days.
 
 
 PPDB
 ----
 
-There was not much discussion or decisions about what exactly PPDB is going to be, but general assumption was that this will be a relational database, most likely PostgreSQL, so that it can support arbitrary queries.
-The amount of data in PPDB will be very large and ever growing as more data is generated by the AP pipeline.
-It is unlikely that a single PostgreSQL server can handle arbitrary requests from multiple clients on a scale of data volume that PPDB is supposed to contain.
+There has been little discussion or decision-making regarding the exact nature of the PPDB architecture.
+The general assumption has been that it will be implemented using a relational database, most likely PostgreSQL, in order to support arbitrary queries.
+The volume of data in the PPDB will be large and ever-growing based on ingestion from the AP pipeline output.
+A single PostgreSQL server will almost certainly be inadequate for handling requests from multiple clients at this scale in a timely manner.
```

Comment:
Maybe replace "requests" with "arbitrary queries". Simple requests can probably be handled OK, but complex queries will need something completely different.